example_buffer.py

```diff
@@ -178,7 +179,12 @@ def _determine_chunk_to_make(write_dir):
     return chunk_to_make, False
 
 def get_window_size(chunk_num):
-    return min(500000, (chunk_num + 5) * MINIMUM_NEW_GAMES)
+    """ Adjust the window size by how far we are through a run.
+    At the start of the run, there's a benefit to 'expiring' the completely
+    random games a little sooner, and scaling up to the 500k game window
+    specified in the paper.
+    """
+    return min(500000, (chunk_num + 5) * (AVG_GAMES_PER_MODEL/2))
```

Comment:
this shuold be integer division `//` to avoid breakage when AVG_GAMES_PER_MODEL is odd.